lib/spack/spack/test/spec_list.py

```diff
@@ -171,7 +171,7 @@ def test_spec_list_matrix_exclude(self, mock_packages):
         )
         assert len(result.specs) == 1
 
-    def test_spec_list_exclude_with_abstract_hashes(self, mock_packages, install_mockery):
+    def test_spec_list_exclude_with_abstract_hashes(self, install_mockery):
```

Comment:
Shouldn't we just blanket add the fixture to `TestSpecList`, to be safer on future additions?

Reply:
😆 I debated doing that but thought you had a preference for minimal fixture use. Given the nature of the class it makes sense though.

Reply:
Done.